### ISSUE TITLE
DRAM.py merge_annotations error, not recognizing argument

### ISSUE THREAD

**Issue by STraving** [WrightonLabCSU/DRAM#231]
Hi DRAM team.

Thanks for a great tool!
I'm analyzing viral sequences from metagenomic samples and processing them using DRAMv. I ran DRAM.py distill on each separate sample. I then became aware of the merge_annotations function but  it throws me an error, maybe I misunderstand what the function really is for and using it wrong?
My folder structure is:
/xx/xx/DRAM_test/sample1/sample1_annotations.tsv
/xx/xx/DRAM_test/sample2/sample2_annotations.tsv

my command and the output:
```
DRAM.py merge_annotations -i /work/work/sachia_56data/DRAM_test/* -o /work/work/sachia_56data/DRAM_combined/
usage: DRAM.py [-h]
               {annotate,annotate_genes,distill,strainer,neighborhoods,merge_annotations}
               ...
DRAM.py: error: unrecognized arguments: /work/work/sachia_56data/DRAM_test/Site_9_1-3

```
bottom line, is there a way a to somehow merge all my annotations before I distill them - or merge the distilled output in an easy way? If not that's also okay, I'll make it work :)

Thanks again for a great tool!

Cheers,
Sachia

**Comment by rmFlynn** [comment]
Can you try `DRAM.py "merge_annotations -i /work/work/sachia_56data/DRAM_test/*" -o /work/work/sachia_56data/DRAM_combined/`?

**Comment by rmFlynn** [comment]
The wild card needs to have quotes to pass a string, sometimes, otherwise bash will pass its interpretation of the wild card.

**Comment by STraving** [comment]
Hi Rory,
Thank you! The following worked:
`DRAM.py merge_annotations -i "/work/work/sachia_56data/DRAM_test/*" -o /work/work/sachia_56data/DRAM_combined1/`

Thank you so much for solving this.
Have a nice Friday and weekend!

Cheers,
Sachia
